### PR TITLE
feat:isprime function implemented

### PR DESCRIPTION
**Pull request by manvithalva12**

### CODE REVIEW

File: freedom.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function isprime(a){
```
Comment (albin1616) [inline freedom.js:1]: use function name as camel case

**Review by albin1616 — CHANGES_REQUESTED** [review]

File: freedom.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    if(a%2===0){
```
Comment (Rajharsh2) [inline freedom.js:2]: - Logic behind checking prime is inncorrect. Its checking even or odd.

File: freedom.js (hunk 1 above)
Comment (manvithalva12) [reply freedom.js]: resolved

**Review by albin1616 — APPROVED** [review]

File: freedom.js (hunk 2 above)
Comment (Rajharsh2) [reply freedom.js:2]: lgtm

**Review by Rajharsh2 — APPROVED** [review]

**Pull request merged**